app.js

```diff
@@ -2,6 +2,8 @@ const express = require('express');
 const app = express();
 const port = process.env.PORT || 3000;
 const multer = require('multer');
+const mongoose = require('mongoose');
+const Countrydata = require('./models/countryModel'); //import schema
```

Comment:
Sorry for not being specific enough. the variable `CountryData` is now the Schema of `CountryData`. I'd like the variable name for a schema to be clear and/or follow the conventions you've posted earlier. Calling a database collection `[...]data` is kind of implied. 

Maybe we can call the collection `countries` so the variable for the Schema would called `Country`. Now mongoose uses some logic to find the collection, however we can forgo this behavior by specifying a collection name as the third argument in the `mongoose.model(..)` function. [More on that here](https://stackoverflow.com/questions/7486528/mongoose-force-collection-name#:~:text=What%20mongoose%20do%20is%20that,your%20schemas%20collection%20name%20option.&text=You%20need%20to%20set%20the%20collection%20name%20in%20your%20schema.).

It would look something like this:

```js
const Country = mongoose.model('Country', schema, 'countries');
```

Reply:
I have processed your feedback. The schema has a better name now. 

I've also added that you can add your bucketlist location and why you want to visit it.
You can add data trough a form, and get saved to a database and showed on a new page.